File: Service/MediCare/RMA.Service.MediCare.Contracts/Entities/Medical/PaymentDetails.cs
using System;

namespace RMA.Service.MediCare.Contracts.Entities.Medical
{
    public class PaymentDetails : Common.Entities.AuditDetails
    {
        public int PaymentId { get; set; }
        public string PaymentStatus { get; set; }
        public bool IsReversal { get; set; }
        public string Payee { get; set; }
        public decimal Amount { get; set; }
        public DateTime DateAuthorised { get; set; }
        public DateTime DatePaid { get; set; }
        public string PaymentStatusReason { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/medicare/src/app/medical-invoice-manager/models/payment-details.ts
import { BaseClass } from 'projects/shared-models-lib/src/lib/common/base-class';

export class PaymentDetails extends BaseClass {
    paymentId: number;
    paymentStatus: string;
    isReversal: boolean;
    payee: string;
    amount: number;
    dateAuthorised: string;
    datePaid: string;
    paymentStatusReason: string;
}